lib/jsdom/browser/js-globals.json

```diff
@@ -0,0 +1,60 @@
+[
```

Comment:
Is this file supposed to be here? Should we wait on merging this until #2743 is merged?

Reply:
I completely messed up the rebase... I will fix that tomorrow. But yes, let wait for your PR to get in first.

Reply:
I'm ok landing in either order. If this lands first then #2743 just needs to delete a few extra lines, probably. I'm worried about getting stuck on #2743, although I've made some good progress I guess.

Reply:
Let's wait for #2743 to land, you have made good progress there. I will try to help out on this, this week.